sass/tji/_data.scss

```diff
@@ -0,0 +1,91 @@
+/*--------------------------------------------------------------
```

Comment:
Can we break this file into _charts.scss and _filters.scss -- so it's not so long and so there's less opportunity of collisions when people are styling the explore the data page?

Reply:
Done.